### ISSUE TITLE
-Dtests.jvms no longer work

### ISSUE THREAD

**Issue by henningandersen** [elastic/elasticsearch#43998]
I think this used to work and may be a sideeffect of the switch from junit to gradle parallel test running. Doing:

```
./gradlew --max-workers 16 -Dtests.jvms=1 :server:integTest
```
still runs 4 threads (on my 4 physical core machine) and
```
./gradlew --max-workers 16 -Dtests.jvms=16 :server:integTest
```
also runs 4 threads.

Bonus points for including that jvms is capped by `--max-workers` into documentation.

**Comment by atorok** [comment]
It does seem we dropped `tests.jvms` from the build plugin when setting `maxParallelForks`

**Comment by mark-vieira** [comment]
The `tests.jvms` setting was something that used to be passed directly to the Ant junit task. Do we want to introduce similar setting (or simply leverage this one) to allow setting `maxParallelForks`?

**Comment by atorok** [comment]
I think we should pass it if present like we still do in  [buildSrc/build.gradle] . Not sure we still need to do that there

**Comment by mark-vieira** [comment]
This has been resolved by #44237.
